app/screens/channel_add_members/channel_add_members.js

```diff
@@ -55,7 +55,6 @@ export default class ChannelAddMembers extends PureComponent {
         this.state = {
             adding: false,
             loading: false,
-            profiles: [],
```

Comment:
Render based on props.profilesNotInChannel instead of state.profiles so that the UI updates on profile change.